qutebrowser/browser/commands.py

```diff
@@ -1927,12 +1927,12 @@ def debug_webaction(self, action, count=1):
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
                        maxsplit=0, no_cmd_split=True)
-    def jseval(self, js_code, quiet=False, *,
+    def jseval(self, js_code, file=False, quiet=False, *,
                world: typing.Union[usertypes.JsWorld, int]=None):
         """Evaluate a JavaScript string.
 
         Args:
-            js_code: The string to evaluate.
+            js_code: The string/file to evaluate.
             quiet: Don't show resulting JS object.
             world: Ignored on QtWebKit. On QtWebEngine, a world ID or name to
                    run the snippet in.
```

Comment:
You'll also need to describe `file` here (and add tests like mentioned above :wink:)